internal/service/apprunner/vpc_connector_test.go

```diff
@@ -167,24 +184,27 @@ resource "aws_vpc" "test" {
   }
 }
 
-resource "aws_security_group" "test" {
-  vpc_id = aws_vpc.test.id
-  name   = %[1]q
+resource "aws_subnet" "test" {
+  cidr_block = "10.1.1.0/24"
+  vpc_id     = aws_vpc.test.id
 
   tags = {
     Name = %[1]q
   }
 }
 
-resource "aws_subnet" "test" {
-  cidr_block = "10.1.1.0/24"
-  vpc_id     = aws_vpc.test.id
+resource "aws_security_group" "test" {
+  vpc_id = aws_vpc.test.id
 
   tags = {
     Name = %[1]q
   }
 }
+`, rName)
+}
 
+func testAccVPCConnectorConfig_basic(rName string) string {
+	return testAccVPCConnectorConfig_base(rName) + fmt.Sprintf(`
```

Comment:
You can also use `acctest.ConfigCompose()` here instead of + for combining the strings.

Reply:
Updated to use ConfigCompose